composer.json

```diff
@@ -22,9 +22,8 @@
         "ralouphie/getallheaders": "^2.0.5 || ^3.0.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.4",
-        "http-interop/http-factory-tests": "^0.5",
-        "ext-zlib": "*"
+        "phpunit/phpunit": "^8.1",
+        "http-interop/http-factory-tests": "dev-master"
```

Comment:
@Nyholm there is no tag and no branch alias for phpunit 8 support in http-interop/http-factory-tests yet